packages/instant/src/types.ts

```diff
@@ -8,18 +8,30 @@ export enum AsyncProcessState {
     FAILURE = 'Failure',
 }
 
+export enum OrderProcessState {
+    NONE = 'None',
+    AWAITING_SIGNATURE = 'Awaiting Signature',
+    PROCESSING = 'Processing',
+    SUCCESS = 'Success',
+    FAILURE = 'Failure',
+}
+
 interface RegularOrderState {
-    processState: AsyncProcessState.NONE | AsyncProcessState.PENDING;
+    processState: OrderProcessState.NONE | OrderProcessState.AWAITING_SIGNATURE;
+}
+interface ProcessingOrderState {
+    processState: OrderProcessState.PROCESSING;
+    txnHash: string;
 }
 interface SuccessfulOrderState {
-    processState: AsyncProcessState.SUCCESS;
+    processState: OrderProcessState.SUCCESS;
     txnHash: string;
 }
 interface FailureOrderState {
-    processState: AsyncProcessState.FAILURE;
+    processState: OrderProcessState.FAILURE;
```

Comment:
we can get rid of this optional property here and merge Successful and Failure order states?

Also lets use `txHash` instead of `txnHash` since its in line with how we abbreviate "transaction hash" in other parts of the code base (e.g. `Web3Wrapper`)